frontend/src/components/dashboard/SelectFilter.vue

```diff
@@ -71,7 +71,7 @@ export default {
     labelValue() {
       if (this.multiple) {
         const list = (this.value || []).map((item) => this.processedItems[item].text)
-        const lang = document.querySelector('html').getAttribute('lang')
+        const lang = this.$store.state.lang.language
```

Comment:
Nice. Maybe we should also cut it to just the first two letters? Not sure what Intl does with our scout language variants. That's also why I used the document locale I think

Reply:
Jup, already happened: https://sentry.io/organizations/ecamp/issues/3737450271/
The html lang attribute currently isn't "safe" either.

Reply:
We could process it with this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/language

Reply:
And also take care of the edge case that there is no language (yet) in the Vuex store: https://sentry.io/organizations/ecamp/issues/3735224550/